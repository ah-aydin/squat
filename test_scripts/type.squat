func main() {
    var a = 1 + 2;
    var s = "ofya";
    println("Type of s is:", type(s));
    println("Type of a is:", type(a));
    println(a);
}
