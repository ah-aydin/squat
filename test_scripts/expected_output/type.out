Compiling and running file: test_scripts/type.squat
1 is not 2
false<type Bool>
0
1
2
3
4
5
a
aa
aaa
0<type Int>
